.prettierignore

```diff
@@ -2,3 +2,4 @@
 **/build
 **/public
 pnpm-lock.yaml
+routeTree.gen.ts
```

Comment:
CI was failing due to formatting on the routeTree.gen.ts. So I excluded the file from prettier format, since the file is handled by TSR.